features/dd-sdk-android-session-replay-compose/src/main/kotlin/com/datadog/android/sessionreplay/compose/mappers/ComposeWireframeMapper.kt

```diff
@@ -0,0 +1,148 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sessionreplay.compose.mappers
+
+import android.os.SystemClock
+import androidx.compose.runtime.Composer
+import androidx.compose.runtime.tooling.CompositionGroup
+import androidx.compose.ui.platform.ComposeView
+import com.datadog.android.Datadog
+import com.datadog.android.api.InternalLogger
+import com.datadog.android.api.feature.FeatureSdkCore
+import com.datadog.android.sessionreplay.SessionReplayPrivacy
+import com.datadog.android.sessionreplay.compose.internal.data.ComposeContext
+import com.datadog.android.sessionreplay.compose.internal.data.ComposeFields
+import com.datadog.android.sessionreplay.compose.internal.data.ComposeWireframe
+import com.datadog.android.sessionreplay.compose.internal.data.UiContext
+import com.datadog.android.sessionreplay.compose.internal.findComposer
+import com.datadog.android.sessionreplay.compose.internal.getSubComposers
+import com.datadog.android.sessionreplay.compose.internal.mappers.CompositionGroupMapper
+import com.datadog.android.sessionreplay.internal.AsyncJobStatusCallback
+import com.datadog.android.sessionreplay.internal.recorder.MappingContext
+import com.datadog.android.sessionreplay.internal.recorder.mapper.BaseWireframeMapper
+import com.datadog.android.sessionreplay.model.MobileSegment
+
+class ComposeWireframeMapper(
+    val privacy: SessionReplayPrivacy
+) : BaseWireframeMapper<ComposeView, MobileSegment.Wireframe>() {
+
+    private val mappers = mapOf<String, CompositionGroupMapper>(
+        // TODO: RUM-0000 Implement mappers for different Composable groups
+        // "Text"
+        // "Button"
+        // "TabRow" : holds the tab row bg color
+        //  "Tab": holds selected tab info
+    )
+
+    override fun map(
+        view: ComposeView,
+        mappingContext: MappingContext,
+        asyncJobStatusCallback: AsyncJobStatusCallback
+    ): List<MobileSegment.Wireframe> {
+        val density = mappingContext.systemInformation.screenDensity.let { if (it == 0.0f) 1.0f else it }
```

Comment:
when this is happening ? Do we have to apply this workaround also for the other mappers ?

Reply:
Indeed, it doesn't happen, I think during some of my test I was trying some stuff and it remained.